straxen/plugins/event_patternfit.py

```diff
@@ -0,0 +1,410 @@
+import strax
+import straxen
+import numpy as np
+from scipy.special import betainc, gammaln, loggamma
+import warnings
+export, __all__ = strax.exporter()
+
+
+@export
+@strax.takes_config(
+    strax.Option('s1_optical_map', help="S1 (x, y, z) optical/pattern map.",
+                 default="XENONnT_s1_xyz_patterns_LCE_corrected_qes_MCva43fa9b_wires.pkl"),
+    strax.Option('s2_optical_map', help="S2 (x, y) optical/pattern map.",
+                 default="XENONnT_s2_xy_patterns_LCE_corrected_qes_MCva43fa9b_wires.pkl"),
+    strax.Option('s1_aft_map', help="Date drive S1 area fraction top map.",
+                 default='s1_aft_dd_xyz_XENONnT_Kr83m_41500eV_25May2021.json'),
+    strax.Option('MeanPEperPhoton', help="Mean of full VUV single photon responce",
```

Comment:
please try to be consistent with capital and lower case / snake_case vs. CamelCase

Reply:
changed everything to lower case